app/frontend/styles/chakra-overrides.scss

```diff
@@ -76,7 +76,7 @@
 
 // Fix the down arrow icon position on the login page accordians
 .chakra-accordion__item {
-  button {
+  button.chakra-accordion__button {
```

Comment:
Only use this chakra overrides scss when absolutely. Please use theme overrides if you must style something globally
https://v2.chakra-ui.com/docs/styled-system/component-style